.pipelines/build-and-push-images.yml

```diff
@@ -15,13 +23,30 @@ jobs:
   - template: ./templates/template-az-cli-login.yml
     parameters:
       azureDevOpsJSONSPN: $(aro-v4-ci-devops-spn)
+
+
   - template: ./templates/template-push-images-to-acr.yml
     parameters:
       rpImageACR: $(RP_IMAGE_ACR)
+
   - template: ./templates/template-az-cli-logout.yml
   - script: |
       cp -a --parents aro "$(Build.ArtifactStagingDirectory)"
     displayName: Copy artifacts
   - task: PublishBuildArtifacts@1
     displayName: Publish Artifacts
     name: aro_deployer
+
+  - script: |
+      set -xe
+      MESSAGE="$(git for-each-ref refs/tags/$TAG --format='%(contents)')"
+      echo "$MESSAGE" > $(System.DefaultWorkingDirectory)/release.txt
```

Comment:
Extracts the annotation from the annotated tag. This is later used as input text for summary of the changelog.
The text have to be stored as a file as azp task can only load file.